src/lite-yt-embed.js

```diff
@@ -102,7 +102,7 @@ class LiteYTEmbed extends HTMLElement {
         const iframeHTML = `
 <iframe width="560" height="315" frameborder="0"
   allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen
-  src="https://www.youtube.com/embed/${this.videoId}?autoplay=1"
+  src="https://www.youtube.com/embed/${this.videoId}?autoplay=1&start=${this.startTime}"
```

Comment:
I kinda feel icky about adding parameters that may not even have a value to the URL. I think I'd wait a bit on pulling this in until my work on actively listening for attribute changes is done this weekend. Once that lands we can do a small refactor of the URL where it is created using the URL object allowing for trivial dynamic creation of the string based on the present attributes. Then supporting any given attributes YouTube allows should be a very thin layer to add on top in a repeatable pattern.